docs/odh/oci-model-storage.md

```diff
@@ -0,0 +1,208 @@
+# Using OCI containers for model storage
+
+Starting ODH 2.18, the ability to use OCI containers as storage for models
+is enabled in KServe by default. The benefits of using OCI containers for
+model storage are described in [the upstream KServe project
+documentation](https://kserve.github.io/website/latest/modelserving/storage/oci/),
+which also describes how to deploy models from OCI images. 
+
+This page offers a guide similar to the upstream project documentation, but
+focusing on the OpenDataHub and OpenShift characteristics. To demonstrate
+how to create an OCI image, the publicly available [MobileNet v2-7
+model](https://github.com/onnx/models/tree/main/validated/vision/classification/mobilenet)
+is used. This model is in ONNX format.
+
+The ODH projects provides configurations for the OpenVINO model server, which
+supports models in ONNX format. Thus, this guide will use this model server
```

Comment:
Maybe it is worth mentioning that OpenVINO does not support all ONNX formats, if I am not mistaken, only DL.

Reply:
OpenVino supports ONNX. @Xaenalt please confirm that.

Reply:
> Maybe it is worth mentioning that OpenVINO does not support all ONNX formats

I don't know. This doc is not meant to be detailed about OpenVINO.